src/style.css

```diff
@@ -338,10 +338,22 @@ button {
   cursor: pointer;
 }
 
+button.red {
```

Comment:
**suggestion:** did you have a look at https://picturepan2.github.io/spectre/elements/buttons.html#buttons-colors?